x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_header.scss

```
@@ -0,0 +1,26 @@
+.appSearchResultHeader {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.appSearchResultHeaderColumn {
+  display: flex;
+  flex-wrap: wrap;
+
+  &--left {
+    justify-content: flex-start;
+
+    .appSearchResultHeaderItem {
+      margin-right: $euiSizeL;
+    }
+  }
+
+  &--right {
+    justify-content: flex-end;
+
+    .appSearchResultHeaderItem + .appSearchResultHeaderItem {
+      margin-left: $euiSizeL;
+    }
+  }
```

I'm reviewing these classes in the Do now and I'm not a super huge fan of their current naming vs. actual behavior in the UI (Side note: SUPER huge kudos to your library route/component for helping w/ this!!)

My main beef is that the `--right` column is actually on the **left** when the `--left` column isn't present, so this feeling like a confusing and untrue name. My preference would either name the columns semantically (e.g., `.appSearchResultHeaderItem--score` or `.appSearchResultHeaderItem--details`) - or even better, I actually don't even see the need at all to distinguish between different types of columns. Flex's `space-between` is doing all the alignment work for us, so we could easily and simply do this instead (given that we don't see ourselves adding yet another column in the future):

```suggestion
  .appSearchResultHeaderItem + .appSearchResultHeaderItem,
  & + & .appSearchResultHeaderItem:first-child {
    margin-left: $euiSizeL;
  }
```

Let me know what you think!

---

Diff for easy reference/cherry-picking: https://github.com/JasonStoltz/kibana/commit/30b21edbf248fab8bd5a3cf023bf54b7fb27063a

I also realized after actually QAing it in-browser that `.appSearchResultHeaderColumn` is actually supposed to be a direct child element of `.appSearchResultHeader`, meaning that it should actually be `.appSearchResultHeader__column`. Apologies for going back and forth on that, my initial review on Friday was in a CSS vacuum/without pulling down the code to inspect the generated DOM - I definitely should have looked closer. That change is in the diff above as well